privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -0,0 +1,127 @@
+                </div>
+
+                <div class="c-premium-onboarding-content">
+                    <div class="c-premium-onboarding-content-image">
+                        <img src="{% static 'images/dashboard-onboarding/woman-email.svg' %}" alt="Woman next to large envelope">
+                    </div>
+                    <div class="c-premium-onboarding-content-description">
+                        <p><strong>Reply to emails without giving away your real address:</strong> Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <p><strong>Get the Relay extension for Firefox</strong> The Relay extension for Firefox makes using email aliases even easier. Reply to emails without giving away your real address: Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <a href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay&utm_medium=banner&utm_campaign=install-addon" class="c-premium-onboarding-install-addon mzp-c-button mzp-t-product">Get Relay extension</a>
+                        <div class="c-premium-onboarding-addon-installed">
+                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
+                            Relay extension installed!
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </li>
+    </ul>
+    <div class="c-premium-onboarding-progress">
+        <div class="mzp-l-content mzp-t-content-xl">
+            <div class="c-premium-onboarding-actions-wrapper">
+                <!-- Step 1 -->
+                <div class="c-premium-onboarding-actions c-premium-onboarding-action-step-1  is-visible">
+                    <button class="mzp-c-button mzp-t-product js-premium-onboarding-next-step">Get Started</button>
+                </div>
+                <!-- Step 2 -->
+                <div class="c-premium-onboarding-actions c-premium-onboarding-action-step-2 ">
+                    <button class="c-button-as-text js-premium-onboarding-next-step" >I'll register my domain later</button>
+                    <button class="mzp-c-button mzp-t-product js-premium-onboarding-next-step is-hidden">Continue</button>
+                </div>
+                <!-- Step 3 -->
+                <div class="c-premium-onboarding-actions c-premium-onboarding-action-step-3 ">
+                    <button class="c-button-as-text js-premium-onboarding-quit-step">Continue, I’ll download the extension later</button>
+                    <button class="mzp-c-button mzp-t-product js-premium-onboarding-quit-step">Create my first email alias</button>
+                </div>
+            </div>
+    
+
+            <ul class="c-premium-onboarding-progress-bar js-premium-onboarding-progress-bar">
```

Comment:
This should probably be an ordered list too, and there should probably be an accessible representation of what these represent (i.e. steps, possibly already completed) for screen readers.

Reply:
`<ol>` Revised in e4ab4ee8f06e1a61deaf6de33cafd6be95d0e3de

What should I do for screen readers?